x-pack/filebeat/filebeat.reference.yml

```
@@ -102,6 +102,15 @@ filebeat.modules:
     # AWS SQS queue url
     #var.queue_url: https://sqs.myregion.amazonaws.com/123456/myqueue
 
+    # AWS S3 bucket
+    #var.bucket: 'arn:aws:s3:::mybucket'
```

Not important, and not related to this PR, but did we consider to use the bucket name directly?
```suggestion
    #var.bucket: 'mybucket'
```
Or name the variable `bucket_arn`, to make it clearer that an arn is expected here?
```suggestion
    #var.bucket_arn: 'arn:aws:s3:::mybucket'
```

---

we need the full `ARN` for metadata.
If we want to rename `bucket` to `bucket_arn` we have to do now before releasing the feature, for me there's no difference, what do you prefer? (it has to be changed in several place anyway)